adapter/src/main/java/com/twilio/kudu/dataloader/DataLoader.java

```diff
@@ -263,67 +276,95 @@ private void bindValues(PreparedStatement statement, UniformLongValueGenerator t
   }
 
   public void loadData(final Optional<Integer> numRowsOverrideOption) {
+    logger.info("scenario start timestamp {}", new Date(scenarioStartTimestamp));
+    logger.info("scenario end timestamp {}", new Date(scenarioEndTimestamp));
```

Comment:
Can we squish these two log statements into one.

Reply:
Done.